src/transformer/events/mod_quiz/attempt_submitted/handler.php

```diff
@@ -23,9 +23,12 @@
 
 function handler(array $config, \stdClass $event) {
     $repo = $config['repo'];
-    $questionattempts = $repo->read_records('question_attempts', [
-        'questionusageid' => $event->objectid
-    ]);
+    $sql = "SELECT qat.* FROM {quiz_attempts} as qa
```

Comment:
I would probably replace this with calls to `$repo->read_record` to simplify tests, it's also likely that we'd want to put some reference to the question usage and question attempt in the statement in future changes.

Reply:
removed as requested.